InnerEye/ML/utils/io_util.py

```diff
@@ -412,24 +412,43 @@ def load_image_in_known_formats(file: Path,
         raise ValueError(f"Unsupported image file type for path {file}")
 
 
-def load_labels_from_dataset_source(dataset_source: PatientDatasetSource, check_exclusive: bool = True) -> np.ndarray:
+def load_labels_from_dataset_source(dataset_source: PatientDatasetSource, check_exclusive: bool = True,
+                                    mask_size: Optional[Tuple[int]] = None) -> np.ndarray:
     """
     Load labels containing segmentation binary labels in one-hot-encoding.
     In the future, this function will be used to load global class and non-imaging information as well.
 
+    :type mask_size: Image size, tuple if integers.
```

Comment:
```suggestion
    :type mask_size: Image size, tuple of integers.
```

Reply:
Please document what mask_size is used for. Consider renaming it to image_size, because a mask can be missing.